### DISCUSSION TITLE
Issue with connecting to RavenDB instance using certificate on a Mac

### DISCUSSION THREAD

**Discussion by MichaelHorn-BuyersEdge** [ravendb/ravendb#12905] [Q&A]
I have been provided a certificate that was generated by RavenDB. I install that certificate on my machine. I go to the RavenDB link, and a list of certificates comes up to choose. The certificate I just installed is not in the list.

When I inspect the cert on my machine, it says it is untrusted. On a Mac, I set it to be trusted. On a PC I moved it into the trusted store, then back into the personal store and it now is trusted.

NB: I did the same thing on my PC with the same cert and got the same results.

**Comment by ayende** [comment]
I assume that you mean from the browser? If so, which browser are you using?
What is not trusted? The server certificate, or are you unable to select the right cert rom the key chain?

**Reply by MichaelHorn-BuyersEdge** [reply]
On the Mac it is Chrome Browser, same with PC.
I was given a zip file containing 3 files, .key, .crt, .pfx. The .pfx is password protected and I was not given that password. I was told that the server generates that password... not sure about that one. I installed the .crt file but it does not show up in the list of certificates that show up when I first navigate to the RavenDB server.

**Reply by MichaelHorn-BuyersEdge** [reply]
Update: I was able to get a new pfx with a password. The browser saw that new certificate, but is now asking for access to the system keychain with admin access. None of my credentials work here.

**Reply by ayende** [reply]
When you generate a certificate from RavenDB, you provide it with the name and password.
Then we generate a certificate and outputs that in KEY, CRT and PFX formats.
If you are working with CRT files, you *must* also register the key. 

If you installed the PFX, you need to provide your own system credentials to register in the keychain.
